expressions/equality.md

```diff
@@ -0,0 +1,70 @@
+# Equality in Pony
+
+Pony features two forms of equality: by value and by identity. 
+
+## Identiy equality
+
+Identity equality checks in Pony are done via the `is` keyword. `is` verifies that the two items are the same. Primitives are always equal to 
+
+```pony
+if None is None then
+  // TRUE!
+  // There is only 1 None so the identity is the same
+end
+
+let a = Foo("hi")
+let b = Foo("bye")
```

Comment:
These two examples would presumably also fail a structural equality check (`==`), since the given string is different for each.  So I don't think it's a good case to demonstrate the difference between structural equality and identity equality (`is`) checking.  It would probably be a good idea to show two objects that are structurally equal, but have different identities.